docs/concepts/credentials/1_ctypes.md

```diff
@@ -26,7 +26,7 @@ When making a claim for a CType, all the following properties are required:
   - Time format e.g. T18:25:43.511Z
   - URI format e.g. https://www.example.com
 
-```js
+```js title="CType schema example"
```

Comment:
What happened in the workshop and getting started in the past, is that these snippets become outdated. Is there a way to test also these simple JSON objects, perhaps by putting them into a script run by the CI and then simply embedded here, like we do for code snippets?

Reply:
I removed most of them